src/output/devtools.html

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<html lang="en">
+  <meta charset="utf-8" />
+  <title>DevTools</title>
+  <style>
+    @media (prefers-color-scheme: dark) {
+      body {
+        background-color: rgb(41 42 45);
+      }
+    }
+  </style>
+  <meta name="referrer" content="no-referrer" />
+  <script
+    type="module"
+    src="https://cdn.jsdelivr.net/npm/chii/public/front_end/entrypoints/chii_app/chii_app.js"
```

Comment:
This would make this feature require a network connection.

Currently the repl doesn't use CDN unless the user is selecting a published version of Vue. If a feature requires internet connection then it should be made opt-in.

Reply:
I added a `chii` option in `Repl` to replace `origin` when deploying to load `chii` resources. A small plugin for vite has also been added to load the value for `chii`

```ts
import VitePluginChii from '@vue/repl/plugin-chii'
```

```tsx
import chii from 'virtual:chii'

function App() {
  return <Repl chii={chii} ... />
}
```